view/reports/report.html

```diff
@@ -369,6 +369,27 @@ <h2><%- result.level.charAt(0).toUpperCase() + result.level.substring(1).toLower
                     )
                     .join(', ')
                   %> <code>#<%- result.code %></code></p>
+                <p>BIK BITV Prüfschritt:
+                    <% _.map(
+                        result.noteCodes,
+                        function (noteCode) {
+                            _.map(
+                                bikCodeMap,
+                                function (bikCodeMapItem) {
+                                    if (bikCodeMapItem.noteCode === noteCode) {
+                                        _.map(
+                                            bikCodeMapItem.bikTestSteps,
+                                            function (bikTestStepsItem) { %>
```

Comment:
In order to be consistent with the rest of the file, please return a string instead of closing the `<% … %>`, thanks.

Reply:
I tried it with the Return statement, but it returns nothing

do you know why?

```javascript
BIK BITV Prüfschritt:
                    <% _.map(
                        result.noteCodes,
                        function (noteCode) {
                            _.map(
                                bikCodeMap,
                                function (bikCodeMapItem) {
                                    if (bikCodeMapItem.noteCode === noteCode) {
                                        _.map(
                                            bikCodeMapItem.bikTestSteps,
                                            function (bikTestStepsItem) {
                                                return '<a href="' + bikTestStepsItem.url + '"> + bikTestStepsItem.name  + '</a>';
                                             }
                                        )
                                    }

                                }
                            )
                        }
                    )%>
```